geps/gep-713.md

```diff
@@ -560,50 +560,536 @@ ties:
+
+Cons:
+
+- Implementations would have to have a webhook, which is another thing to run.
+- The webhook will need to have the same data model that the implementation uses,
+  and keep track of which GatewayClasses, Gateways, Routes, and Policies are
+  relevant. Experience suggests this will not be a trivial engineering exercise,and will add a lot of implementation complexity.
+
+#### `kubectl` plugin or command-line tool
 To help improve UX and standardization, a kubectl plugin will be developed that
 will be capable of describing the computed sum of policy that applies to a given
 resource, including policies applied to parent resources.
 
 Each Policy CRD that wants to be supported by this plugin will need to follow
-the API structure defined above and add a `gateway.networking.k8s.io/policy:
-true` label to the CRD.
-
-### Status
-
-In the current iteration of this GEP, metaresources and Policy objects don't
-have any standard way to record what they're attaching to, or applying settings
-to in the case of Policy Attachment. There are some recommended Condition types
-defined below, but further work on the status design is required to ensure that
-some problems are resolved:
-
-* When multiple controllers are implementing the same Route and recognize a
-  policy, it must be possible to determine which controller was
-  responsible for adding that policy reference to status. Adding Conditions to
-  status on the Policy instead can be helpful here, but we're still lacking a way
-  for the Route or Gateway owner to find _all_ the Policies that are influencing
-  their object.
-* For this to be somewhat scalable, we must limit the number of status updates
-  that can result from a metaresource update.
-* Since we only control some of the resources a policy might be attached to,
-  adding policies to status would only be possible on the policy objects themselves
-  or on Gateway API resources, not Services or other kinds of backends.
-
-Previous experience in the Kubernetes API has made it clear that having a single
-object that can cause status updates to occur across many other objects can have
-a big performance impact, so the status design must be very carefully done to
-avoid these kind of fanout problems.
-
-However, the whole purpose of having a standardized Policy API structure and
-patterns is intended to make this problem solvable both for human users and with
-tooling.
-
-This is currently a _very_ open question. A discussion is ongoing at
-[#1531](https://github.com/kubernetes-sigs/gateway-api/discussions/1531), and this
-GEP will be updated with any outcomes.
+the API structure defined above and add a `gateway.networking.k8s.io/policy: true`
+label to the CRD.
 
 ### Conditions
-Controllers using the Gateway API policy attachment model SHOULD populate the 
+
+Implementations using Policy objects MUST include a `spec` and `status` stanza, and the `status` stanza MUST contain a `conditions` stanza, using the standard Condition format.
```

Comment:
I may just be missing it, but I'm not finding any requirement in this GEP for implementations to set the `Accepted` condition on policy objects they are implementing. 

https://github.com/kubernetes-sigs/gateway-api/blob/f638d3fe75a69354b4a15b6c9e9e475c4e7d8062/apis/v1alpha2/policy_types.go#L51-L64

With that said, this also becomes problematic if multiple controllers are implementing the same policy resource, so maybe we need to adjust the guidance here to leave room for `conditions` stanzas to be nested within a `parents` list similar to what we do for Routes?

Reply:
It was already there, but I've updated to clarify a bit.